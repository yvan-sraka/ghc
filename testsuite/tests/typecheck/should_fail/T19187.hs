{-# LANGUAGE UndecidableInstances #-}

module T19187 where

data T

instance Eq Int => Eq T
