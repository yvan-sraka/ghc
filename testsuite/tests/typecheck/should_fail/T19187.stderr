
T19187.hs:7:10: error:
    • Non type-variable argument in the constraint: Eq Int
      (Use FlexibleContexts to permit this)
    • In the instance declaration for ‘Eq T’
